app/controllers/hosts_controller.rb

```diff
@@ -431,6 +432,30 @@ def update_multiple_owner
     redirect_back_or_to hosts_path
   end
 
+  def select_multiple_power_state
+  end
+
+  def update_multiple_power_state
+    # simple validations
+    if (params[:power].nil?) or (action=params["power"]["action"]).nil? or !PowerManager::REAL_ACTIONS.include?(action)
```

Comment:
Move this params validation to a `before_filter` that applies to this method only